### ISSUE TITLE
NVDA does not announce links imbedded inside of a heading with a title set on the heading tag

### ISSUE THREAD

**Issue by bug-reporter-1** [nvaccess/nvda#6363]
I have seen examples of this on the internet where NVDA does not read links in headings. I have created a simple HTML example to demonstrate the problem. The attached code sample works correctly with Orca and Firefox so I believe this is a problem with NVDA as opposed to Firefox.

**Comment by bug-reporter-1** [comment]
``<h2 title="test">
<a href="http://www.google.com">Google Home Page</a>
</h2>

**Comment by bug-reporter-1** [comment]
[example.txt](https://github.com/nvaccess/nvda/files/464024/example.txt)

**Comment by ehollig** [comment]
This works for me as expected. Closing as worksforme
